tests/test_config.py

```diff
@@ -502,6 +502,17 @@ def test_non_ascii(self):
         self.assertEqual(cov.config.exclude_list, ["first", "✘weirdo", "third"])
         self.assertEqual(cov.config.html_title, "tabblo & «ταБЬℓσ» # numbers")
 
+    def test_json_config(self):
+        self.make_file(".coveragerc", """\
+            [json]
+            pretty_print = True
+            show_contexts = True
+            """)
+        cov = coverage.Coverage()
+
+        self.assertEqual(cov.config.json_show_contexts, True)
+        self.assertEqual(cov.config.json_pretty_print, True)
+
```

Comment:
The other config sections don't have special-purpose test functions. I tend to just add new settings into LOTSA_SETTINGS.  But I could be convinced.

Reply:
addressed, long live LOTSA_SETTINGS